docs/de/i-doit-pro-add-ons/documents/index.md

```diff
@@ -20,6 +20,72 @@ Die Einbindung dynamischer Inhalte der Dokumentation erfolgt mit Platzhaltern. H
 
 Erstellte Vorlagen können genutzt werden, um für Objekte in der Dokumentation Dokumente zu erstellen. In diesem Bereich steht eine Beschreibung zur Verfügung, die ausführlich auf die Erstellung, Aktualisierung und den Export dieser Dokumente eingeht.
 
+## CLI Console Commands und Optionen
+
+| Command     | Interne Beschreibung                                                     |
+| ----------- | ------------------------------------------------------------------------ |
+| [compile-documents](#compile-documents) | Erlaubt es dir ein oder mehrere Dokumente zu kompilieren |
+| [documents](#documents) | Dokumente und Revisionen erzeugen und exportieren |
```

Comment:
Does relate to the first title on the top, try documents_1
Also in nagios you added a admonition, why not here?

Reply:
documents-1 worked :p